code/src/java/pcgen/gui2/SplashScreen.java

```diff
@@ -44,6 +46,7 @@
 public class SplashScreen extends JWindow implements PCGenTaskListener
 {
 
+	private static final long serialVersionUID = 5186500840685769783L;
```

Comment:
mostly it shuts up some warnings from PMD. Otherwise its not useful. I guess I could go and try and remove the warning generator instead.
It might also be correct to make the window not serializable.